src/Razor/test/Microsoft.AspNetCore.Razor.Language.Test/IntegrationTests/ComponentTypingTest.cs

```diff
@@ -88,9 +88,45 @@ public void Regression_1068()
 @functions {
     Test.ModelState ModelState { get; set; }
 }
-");
+", throwOnFailure: false);
```

Comment:
Without this, the typing test would fail during the first few iteration because we generate a RazorDiagnostic for incomplete things and we don't want that to fail the test.